test/ClangImporter/foreign_errors.swift

```diff
@@ -14,7 +14,7 @@ func test0() {
 func testAndReturnError() throws {
   try ErrorProne.fail()
   try ErrorProne.go()
-  try ErrorProne.tryAndReturnError() // collides with 'try' keyword
+  try ErrorProne.tryAndReturnError(()) // collides with 'try' keyword
```

Comment:
We add a dummy argument when there's a naming conflict with another method, but we shouldn't when there's a conflict with a keyword.